lib/DDG/Spice/Nutrition.pm

```diff
@@ -24,7 +39,12 @@ handle query_lc => sub {
     if (/^(.+?)\s?$question_regex$attribute_regex(?:\?)?/) {
         return $1 if $1;
     }
-
+    
+    # check for matching fruits, vegetables, animal products, meat, ...
+    if ( grep( /^$_$/, @food_triggers ) ) {
```

Comment:
@fraed perhaps we should try a naïve approach to handling plurals? If this fails, let's try removing a trailing `s` or replacing `ies` with `y`?